src/20_Components/amp-list.html

```diff
@@ -25,6 +25,38 @@
     bottom: 0;
     right: 0;
   }
+  <!--
+    #### CSS for the amp-list loader
```

Comment:
Commenting inside the css block is unfortunately not supported (but would be very nice to have).  We have to add the docs before the `<style amp-custom>...`.